setup.py

```diff
@@ -26,4 +42,10 @@
       },
       install_requires=[
             'numpy>=1.9.2'
-      ])
+      ],
+      classifiers=[
+        "License :: OSI Approved :: Apache Software License",
```

Comment:
Is there a way to specify Apache License v2? Not familiar with licenses so not sure how important that is

Reply:
That is the only Apache classifier. 
I'll just change it back to using the license keyword instead of classifier

Reply:
openMDAO seems to do both: https://github.com/OpenMDAO/OpenMDAO/blob/master/setup.py

Maybe that's a worthwhile approach? Not sure what's best here, so your call.